src/Model/GlobalCommands.re

```diff
@@ -0,0 +1,390 @@
+open Actions;
+open Feature_Commands.Schema;
+
+let copyFilePath =
+  define(
+    ~category="Editor",
+    ~title="Copy Active Filepath to Clipboard",
+    "copyFilePath",
+    CopyActiveFilepathToClipboard,
+  );
+
+let acceptSelectedSuggestion =
+  define("acceptSelectedSuggestion", Command("acceptSelectedSuggestion"));
+
+let selectPrevSuggestion =
+  define("selectPrevSuggestion", Command("selectPrevSuggestion"));
+
+let selectNextSuggestion =
+  define("selectNextSuggestion", Command("selectNextSuggestion"));
+
+let undo = define("undo", Command("undo"));
+let redo = define("redo", Command("redo"));
+
+let indent = define("indent", Command("indent"));
+let outdent = define("outdent", Command("outdent"));
+
+module Editor = {
+  module Action = {
+    let detectIndentation =
+      define(
+        ~category="Editor",
+        ~title="Detect Indentation from Content",
+        "editor.action.detectIndentation",
+        Command("editor.action.detectIndentation"),
+      );
+
+    let clipboardPasteAction =
+      define(
+        "editor.action.clipboardPasteAction",
+        Command("editor.action.clipboardPasteAction"),
+      );
+
+    let indentLines =
+      define(
+        "editor.action.indentLines",
+        Command("editor.action.indentLines"),
+      );
+
+    let outdentLines =
+      define(
+        "editor.action.outdentLines",
+        Command("editor.action.outdentLines"),
+      );
+  };
+};
+
+module List = {
+  let focusDown = define("list.focusDown", ListFocusDown);
+  let focusUp = define("list.focusUp", ListFocusUp);
+
+  let select = define("list.select", ListSelect);
+  let selectBackground =
+    define("list.selectBackground", ListSelectBackground);
+};
+
+module Oni = {
+  module Explorer = {
+    let toggle =
+      define(
+        ~category="View",
+        ~title="Toggle File Explorer visibility",
+        "oni.explorer.toggle", // use workbench.action.toggleSidebarVisibility instead?
```

Comment:
There are several instances where we've invented our own commands when there already exists vscode commands that essentially reflects the same intent. I'll do another PR to clean this up later, to try to contain this one as much as possible.

Reply:
Thanks @glennsl !